docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -94,46 +88,40 @@ corresponding CDAP types.
 | Array                  | array     |
 | Record                 | record    |
 
-Limitations
+Pagination
 ----------
 
 Zendesk plugin supports two types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/) and [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
-Offset and Time-Based Pagination might result in data duplication.
-
-### Data Duplication issues with Zendesk APIs
-
-The Zendesk Multi Batch Source plugin might return duplicate records for offset and time-based exports as mentioned in Zendesk docs.
-[Zendesk documentation](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#excluding-duplicate-items)
-
 Objects that support both offset and time-based exports:
```

Comment:
This can be removed as it is already there in table

Reply:
Duplicate issue is not defined in table. Do u mean to remove the example which is mentioned above to this, bcoz it is mentioned in the table.

Reply:
No Object list you need to remove.

Reply:
done